src/main/kotlin/net/maizegenetics/phgv2/cli/CreateFastaFromHvcf.kt

```diff
@@ -148,20 +150,20 @@ class CreateFastaFromHvcf : CliktCommand( help = "Create a fasta file from a hvc
         val ranges = rangesAndOtherInfo.flatMap { it.first }
         val seqs = retrieveAgcContigs(dbPath,ranges)
 
-        return rangesAndOtherInfo.map { it.third.copy(sequence = buildHapSeq(seqs, it.second,it.third)) }
+        return rangesAndOtherInfo.map { it.third.copy(sequence = buildHapSeq(seqs, sampleName, it.second,it.third)) }
```

Comment:
Rereading this, I realize I might have coded this up wrong. I am not sure what will happen if the sample Name being passed in the sample name for an imputed path.  This would mean that there would need to be potentially multiple different assembly sampleNames which should correspond to the sample name in the Variant context/ALT header.  It works correctly for the assemblies, but may have some trouble with paths.  I need to think through this.  AGC retrieval might be very different here as well based on some of the issues we have run into.